hack/ocm-minikube.sh

```diff
@@ -0,0 +1,309 @@
+#!/bin/sh
+# shellcheck disable=SC2046,SC2086
+
+# open cluster management (ocm) hub and managed minikube kvm amd64 clusters deploy
+# https://github.com/ShyamsundarR/ocm-minikube/README.md
+
+set -x
+set -e
+trap 'echo exit value: $?' EXIT
+
+#minikube delete -p cluster2
+#minikube delete -p cluster1
+#minikube delete -p hub
+
+mkdir -p ${HOME}/.local/bin
+PATH=${HOME}/.local/bin:${PATH}
+
+if ! command -v curl; then
+	wget -O ${HOME}/.local/bin/curl https://github.com/moparisthebest/static-curl/releases/download/v7.76.0/curl-amd64
+	chmod +x ${HOME}/.local/bin/curl
+fi
+
+if ! command -v minikube; then
+	# https://minikube.sigs.k8s.io/docs/start/
+	minikube_version=latest
+	minikube_version=v1.18.1
+	curl -LRo ${HOME}/.local/bin/minikube https://storage.googleapis.com/minikube/releases/${minikube_version}/minikube-linux-amd64
+	unset -v minikube_version
+	chmod +x ${HOME}/.local/bin/minikube
+fi
+
+minikube_start_options=--driver=kvm2
```

Comment:
According to [1] it makes use of some feature gates while starting the minikube cluster. Should they be considered as the minikube_start_options? 

======================================================================================

minikube start --force --memory="4096" --cpus="2" -b kubeadm --kubernetes-version="v1.19.2" --driver="kvm2" --feature-gates="BlockVolume=true,CSIBlockVolume=true,VolumeSnapshotDataSource=true,ExpandCSIVolumes=true" --profile="cluster1"

=========================================================================================

@ShyamsundarR 

[1] https://hackmd.io/239bv_V5QlaXBa1rl3aFAA?both

Reply:
> According to [1] it makes use of some feature gates while starting the minikube cluster. Should they be considered as the minikube_start_options?

No these are not required in this case, we should be fine.

Reply:
Most of these feature gates are already beta and enabled by default in latest kubernetes version.